rfcs/text/0041/asset.yml

```diff
@@ -0,0 +1,174 @@
+---
+- name: asset
+  title: Asset information
+  group: 2
+  short: Fields for describing an asset
+  beta: |
+    These fields are in beta and are subject to change.
+  description: >
+    Fields to describe an organization asset such as a host, user, or
+    an infrastructure. These fields can be nested under other objects that
+    identifies an asset such as host, user, network, and cloud schemas.
+  reusable:
+    top_level: false
```

Comment:
Why should asset not be used on the top level?

Reply:
Probably because they are tied to main object and don't have meaning themselves. But good question

Reply:
@tinnytintin10 agreed. Because of the open source nature of ECS, we should not limit ourselves to the vision of solution (Security & Observability) use cases only, as any Elastic user may desire to map data into the asset schema that we haven't considered.